### DISCUSSION TITLE
Image component is breaking when using skeleton

### DISCUSSION THREAD

**Discussion by gmcamposano** [vercel/next.js#50657] [Help] [answered]
### Summary

The code below seemed to work with other versions of Image, but with next 13 it keeps crashing and not working as expected. 

### Additional information

```js
import ImageCard from '@/imageCard'

export default function Home() {
  return (
    <div className="relative">
      <ImageCard
        src={`https://picsum.photos/seed/500/500`}
        alt="Random image"
        width={500}
        height={500}
      />
    </div>
  )
}

//imageCard component

'use client'
import Image from 'next/image'
import { useState } from 'react'
import Skeleton from 'react-loading-skeleton'
import 'react-loading-skeleton/dist/skeleton.css'
export default async function ImageCard(props) {
  const [loading, setLoading] = useState(true)
  return (
    <>
      {loading && <Skeleton width={props.width} height={props.height} />}
      <Image
        style={{ display: loading ? 'none' : 'block' }}
        src={props.src}
        alt="Random image"
        width={props.width}
        height={props.height}
        onLoadingComplete={() => setLoading(false)}
      />
    </>
  )
}
```

**Comment by devjiwonchoi** [comment, marked as answer]
First of all, your `ImageCard` component is an `Asynchronous` function, which technically returns a `Promise<Element>` type.
Since `Promise<Element>` is not a valid JSX Element, it will cause an error if you use it as a `JSX component`.

> 'ImageCard' cannot be used as a JSX component.
  Its return type 'Promise<Element>' is not a valid JSX element.
    Type 'Promise<Element>' is missing the following properties from type 'ReactElement<any, any>': type, props, key | ts(2786)

It seems like your `ImageCard` component does not need to be async, so I would recommend you remove it.

One last thing to double check is if you have configured the domain `picsum.photos` in your `next.config.js`.

Example:
```javascript
// next.config.js

/** @type {import('next').NextConfig} */
const nextConfig = {
  images: {
    domains: ['picsum.photos'],
  },
};

module.exports = nextConfig

```

In summary, please remove the `async` from your `ImageCard` component and add `picsum.photos` domain to your `next.config.js`.

**Comment by gmcamposano** [comment]
Yes, the problem was that I was making it Async. That fixed the issue.
